Contacts

bogdankol opened the pull request:
там файл спрайта - он не должен конфликтовать, взял я его с ветки dev, но добавил свою иконку правой стрелочки (она отличается от просто правой стрелочки, которую заспрайтила Оля)....добавил цвет текста, жирность и тень

bogdankol committed:
разметка

bogdankol committed:
добавил фон

bogdankol committed:
иконки + разметка кнопок для модалок + правки списка контактов + наметка селекторов

bogdankol committed:
Update _contacts.scss

bogdankol committed:
стили для текста

bogdankol committed:
чуток код поправил + спрайт

bogdankol committed:
убрал демо картинку

bogdankol commented:
че то какая-то фигня с иконкой - не меняется цвет, попробую исправить

KoliadzhynYI committed:
Merge branch 'dev' into contacts

bogdankol committed:
убрал свой свг код - битый код

bogdankol committed:
Merge branch 'contacts' of https://github.com/ITeam-by-GOIT/ice-cream into contacts

AngieBoiko commented on src/sass/base/_common.scss:
```diff
@@ -20,7 +20,7 @@ body {
 }
 
-.list {
+ul {
```
вчера на миттинге договорились оставить класс .list, базовые найстройки не надо самостоятельно менять, только добавляем переменные, после fatch origin и проверки нет ли уже таких

AngieBoiko commented on src/sass/main.scss:
```diff
@@ -12,6 +12,7 @@
 
 /* === Layout === */
 @import './layout/main';
+@import './layout/contacts';
```
паршал _contacts.scss  уже подключен @import './layout/main';

Koliadjun reviewed (changes requested):
Виправ дані зауваження.

AngieBoiko commented on src/sass/layout/_contacts.scss:
```diff
@@ -0,0 +1,16 @@
+.container {
```
у нас уже есть .container, класс закреплен src/sass/base/_container.scss

AngieBoiko reviewed (commented):
поправки по ветке contacts

AngieBoiko commented on src/sass/base/_common.scss:
```diff
@@ -20,7 +20,7 @@ body {
 }
 
-.list {
+ul {
```
@Koliadjun  мы одновременно смотрели код))) написала тоже комментарии по этому поводу)

bogdankol replied:
+++

ghost commented:
![](https://github.trello.services/images/mini-trello-icon.png) [contacts](https://trello.com/c/ol3UP58V/49-contacts)

ikalyniak commented on src/images/contacts_background/section_dots.svg:
```diff
@@ -0,0 +1,9 @@
+<pattern id="pattern0" patternContentUnits="objectBoundingBox" width="1" height="1">
+<use xlink:href="#image0" transform="scale(0.000520833 0.0026178)"/>
+</pattern>
+<image id="image0" width="1920" height="382" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAB4AAAAF+CAMAAABUNPenAAAAeFBMVEUAAAD/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMn/uMkyEbvjAAAAJ3RSTlMA+QTuHZMJAd/WvbLPFw3nhGRRxqpDJRH8bEzzo5t4Xj4uczmMWDO3qtfAAAAG6ElEQVR42uzBgQAAAACAoP2pF6kC ...
```
что это ?

bogdankol replied:
+++

ikalyniak commented on src/partials/contacts.html:
```diff
@@ -0,0 +1,86 @@
+            <li class="contacts__item">
+                <p class="contacts__item-title">cafe</p>
+                <ul class="contacts__item-header">
+                    <li class="item item--black">Chicago</li>
```
модификатор не может существовать без самого элемента. 
contacts =это блок
contacts__item =это элемент блока
contacts__item--black =это модификатор элемента

ikalyniak commented on src/partials/contacts.html:
```diff
@@ -0,0 +1,86 @@
+                </button> 
+            </li>
+            <li class="contacts__modal-item">
+                <button type="button" class="contacts__button">Franchise 
```
универсальный класс как раз и задает правильный фон и цвет, а не просто красный. А паддинги, высота и жирность текста на  секции контактов и в герое  разные поэтому их нужно прописывать самому

bogdankol replied:
+++

ikalyniak commented on src/sass/layout/_contacts.scss:
```diff
@@ -0,0 +1,108 @@
+    max-width: 280px;
+    margin: 0 auto;
+    padding: 0 30px;
+  }
```
мы же вроде используем компоненты, контейнер есть в base

bogdankol committed:
исправление замечаний

замечание №1 - убрал контакты из мейна сцсс
№2 - пустой комментарий убрал
№3 - заменил на list и добавил в html
№4 - классы кнопок использовал из паршала общего
№5 - убрал свой контейнер
№6 - №8 - совпадают с предыдущими
№9 - тот код это был слой фона в виде свг, переделал на пнг и пнг@2х, используется только для пк версии
№10 - исправил названия классов
№11 - использую контейнер из компонентов, но это спорный вопрос, так как Денис говорил без паддингов контейнера

bogdankol commented:
вроде все исправил

bogdankol committed:
поправил цвета в списке контактов - замена на переменные, добавление новой переменной для титулок парных контактов

bogdankol committed:
список контактов для моб версии

bogdankol commented on src/sass/main.scss:
```diff
@@ -12,6 +12,7 @@
 
 /* === Layout === */
 @import './layout/main';
+@import './layout/contacts';
```
+

bogdankol commented on src/sass/utils/_variables.scss:
```diff
@@ -16,4 +16,7 @@
   --bg-social: #fea5bb;
   --bg-header-hero: #ffb8ca;
+
+  // contacts
```
+

bogdankol replied:
=

bogdankol replied:
+++

bogdankol committed:
сделал список растягивающимся до первой точки перелома и меняется размер шрифта при 480

bogdankol committed:
список + кнопки + адаптив

№1 - поправил фон секции (на 2х стоит ограничение макс размера, если не поставить оно отзывчиво растягивается до своего макс размера)
№2 - оптимизация кода
№3 - адаптив для планшета и пк

bogdankol commented on src/sass/layout/_contacts.scss:
```diff
@@ -0,0 +1,108 @@
+.container {
```
+++

bogdankol commented on src/sass/layout/_contacts.scss:
```diff
@@ -0,0 +1,108 @@
+  margin: 0 auto;
+  outline: 1px solid #ff0000;
+}
+.contacts__item-header {
```
+++

bogdankol commented:
вроде все сделал, остались модалки

bogdankol committed:
обновился package - он сам, я ничего не делал,,,, + семантика времени работы

AngieBoiko commented on src/sass/layout/_contacts.scss:
```diff
@@ -0,0 +1,298 @@
+}
+
+.contacts__item-title {
+  background: #f0d1a5;
```
давай я сделаю переменную для этого цвета, он у менят в секции тоже есть, назову --bg-beige-color. Можешь себе ее ставить сразу, при фетче подтянится

bogdankol replied:
окей, добавил var

AngieBoiko commented on src/sass/utils/_variables.scss:
```diff
@@ -7,6 +7,8 @@
   --color-design-black: #000000;
   --color-subtitle: #ffa5ba;
   --color-description: #907e82;
+  --color-even-contacts: #C2E297;
```
этот цвет повторяется у меня в секции, давай его назовем универсально --bg-green-color, я сейчас у себя добавлю в переменные и будем уже сливать ветку, удали пожалуйста у себя из переменны,  в стилях поменяй на новое название, при фетче должно подтянутся без конфликтов

bogdankol replied:
окей, удалил и поменял

bogdankol replied:
пушу

bogdankol committed:
переменную цвета убрал, так как ее Анжела внесет,,, поменял на ее название в сцсс
